maxim/logger/llamaindex/client.py

```
@@ -0,0 +1,422 @@
+    def _check_instrumented():
+        global _INSTRUMENTED
+        if _INSTRUMENTED:
+            scribe().debug("[MaximSDK] LlamaIndex already instrumented")
+            return True
+        _INSTRUMENTED = True
+        return False
+
+    if _check_instrumented():
+        return
+
+    def make_maxim_wrapper(original_method, base_op_name):
```

Ruff (0.12.2)

`make_maxim_wrapper` is too complex (40 > 10)

(C901)